packages/ui-text/src/Text/props.ts

```diff
@@ -57,10 +57,41 @@ type TextOwnProps = {
   elementRef?: (element: Element | null) => void
   fontStyle?: 'italic' | 'normal'
   letterSpacing?: 'normal' | 'condensed' | 'expanded'
-  lineHeight?: 'default' | 'fit' | 'condensed' | 'double'
-  size?: 'x-small' | 'small' | 'medium' | 'large' | 'x-large' | 'xx-large'
+  lineHeight?:
+    | 'default'
+    | 'fit'
+    | 'condensed'
+    | 'double'
+    | 'lineHeight100'
+    | 'lineHeight125'
+    | 'lineHeight150'
+  size?:
+    | 'x-small'
+    | 'small'
+    | 'medium'
+    | 'large'
+    | 'x-large'
+    | 'xx-large'
+    | 'descriptionPage'
+    | 'descriptionSection'
+    | 'content'
+    | 'contentSmall'
+    | 'legend'
   transform?: 'none' | 'capitalize' | 'uppercase' | 'lowercase'
-  weight?: 'normal' | 'light' | 'bold'
+  /**
+   * Sets multiple props at once. (size, fontStyle, lineHeight, weight)
+   * If set, these props are not allowed.
+   * NOTE: this is the recommended way of setting these values
+   */
+  variant?:
+    | 'descriptionPage'
+    | 'descriptionSection'
+    | 'content'
+    | 'contentImportant'
+    | 'contentQuote'
+    | 'contentSmall'
+    | 'legend'
+  weight?: 'normal' | 'light' | 'bold' | 'weightRegular' | 'weightImportant'
```

Comment:
Can you please edit the storybook files? These are generating way too many stories.